### PR TITLE
updating build to function again

### PR DESCRIPTION
**Pull request by luthermonson**

### CODE REVIEW

**Review by rosskirkpat — APPROVED** [review]
LGTM

**Comment by phillipsj** [comment]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -7,64 +7,83 @@ platform:
...
+- name: docker-build
+  image: plugins/docker
   environment:
-    DOCKER_USERNAME:
-      from_secret: docker_username
-    DOCKER_PASSWORD:
+    TAG: ${DRONE_TAG}
+  settings:
+    purge: false
+    custom_dns: 1.1.1.1
```
Comment (briandowns) [inline .drone.yml:27]: Why is this necessary?

File: Makefile [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
 .PHONY: image-manifest
 image-manifest:
-	docker image inspect rancher/hardened-kubernetes:$(TAG)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create rancher/hardened-kubernetes:$(TAG) \
-		$(shell docker image inspect rancher/hardened-kubernetes:$(TAG) | jq -r '.[] | .RepoDigests[0]')
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create --amend \
+		$(ORG)/hardened-kube-proxy:$(TAG) \
+		$(ORG)/hardened-kube-proxy:$(TAG)-$(ARCH)
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push \
+		$(ORG)/hardened-kube-proxy:$(TAG)
```
Comment (briandowns) [inline Makefile:56]: New line

File: .drone.yml (hunk 1 above)
Comment (luthermonson) [reply .drone.yml:27]: something with drone, we have to do it for rancher, not sure why any more tbh just a thing we add. ill ask eio if it's still necessary.

File: Makefile (hunk 2 above)
Comment (luthermonson) [reply Makefile:56]: good catch, i didnt even use this cause i used the manifest plugin in drone but ill fix it for people to test with is

**Review by briandowns — APPROVED** [review]
LGTM
